core/drivers/stm32_i2c.c

```diff
@@ -1151,6 +1152,86 @@ int stm32_i2c_master_transmit(struct i2c_handle_s *hi2c, uint32_t dev_addr,
 	return i2c_write(hi2c, &request, p_data, size);
 }
 
+/*
+ * Optimized 1 byte read/write function for unpaged sequences.
+ * 8-bit addressing mode / single byte transferred / use default I2C timeout.
+ * 'unpg88' refers to 8bit address/8bit value.
+ */
+int stm32_i2c_read_write_membyte(struct i2c_handle_s *hi2c, uint16_t dev_addr,
```

Comment:
Return codes are a bit inconsistent: here you use 0 and 1, other functions in this file use 0 and -1. I would argue that `bool` would be better for this kind of success/failure status (and `TEE_Result` for the more general case). But Perhaps you should just use -1 instead of 1 for consistency.

Reply:
ok, i'll fix the return code